addons/l10n_ch/static/src/scss/report_swissqr.scss

```
@@ -209,6 +209,8 @@ body.l10n_ch_qr, body[data-oe-report="l10n_ch.l10n_ch_qr_report"] {
         height: 105mm * $ratio - $rounding_offset;
         border: 0.75pt * $ratio dashed black;
         border-right: 0pt;
+        border-left: none;
+        border-bottom: none;
```

```suggestion
        border-top: 0.75pt * $ratio dashed black;
```

You may be able to do it like this (shorter).

---

just tested it, it seems to do the trick